website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -516,7 +516,7 @@ and consider if they're appropriate for your deployment.
       Default mode of the mounted files.
 
     ```yaml
-    extraVolumes:
```

Comment:
I don't think we need to change this example, since it's in the `extraVolumes` section.